src/chainparams.cpp

```diff
@@ -805,7 +805,7 @@ class CRegTestParams : public CChainParams {
         consensus.DIP0003EnforcementHeight = 500;
         consensus.DIP0003EnforcementHash = uint256();
         consensus.DIP0008Height = 432;
-        consensus.BRRHeight = 2500; // see block_reward_reallocation_tests
+        consensus.BRRHeight = 1000; // see block_reward_reallocation_tests
```

Comment:
```suggestion
        consensus.BRRHeight = 900; // see block_reward_reallocation_tests
```

Values lower than 900 I would not set because it would be less than DIP0024 and V19. But I see no issues with (not tested by me)